README.md

```diff
@@ -68,17 +68,10 @@ iex> Lens.get_and_map(data, lens, fn size -> {size, round(size)} end)
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed as:
+[Available in Hex](https://hex.pm/packages/lens), the package can be installed as:
 
   1. Add lens to your list of dependencies in `mix.exs`:
 
         def deps do
           [{:lens, "~> 0.0.1"}]
         end
-
-  2. Ensure lens is started before your application:
```

Comment:
Oh, I didn't have releases on mind, but what you mention sounds legit, so let's keep it on the applications list.